src/Databases/DataLake/GlueCatalog.cpp

```diff
@@ -286,27 +325,18 @@ bool GlueCatalog::tryGetTableMetadata(
                 if (column_params.contains("iceberg.field.current") && column_params.at("iceberg.field.current") == "false")
                     continue;
 
-                schema.push_back({column.GetName(), getType(column.GetType(), can_be_nullable)});
-            }
-            result.setSchema(schema);
-        }
-
-        if (result.requiresCredentials())
-            setCredentials(result);
+                String column_type = column.GetType();
+                if (column_type == "timestamp")
```

Comment:
Timetstamp type should be quite popular, so we will fallback to the hacky check very often :(